Source/WebCore/Modules/cookie-store/CookieInit.h

```diff
@@ -35,8 +35,8 @@ namespace WebCore {
 struct CookieInit {
     String name;
     String value;
-    std::optional<DOMHighResTimeStamp> expires;
-    String domain;
+    std::optional<DOMHighResTimeStamp> expires { };
```

Comment:
I believe these do not require explicit initialization

Reply:
This was a required build fix for EWS. I suggested it.

It is *usually* not needed. It is however sometimes needed for POD structs if you want to only initialize some of the data members.

Reply:
ah, I see!